sematic/plugins/storage/s3_storage.py

```diff
@@ -14,13 +14,13 @@
     PluginVersion,
 )
 from sematic.config.settings import get_plugin_setting
-from sematic.plugins.abstract_storage import AbstractStorage, PayloadType, ReadPayload
+from sematic.plugins.abstract_storage import AbstractStorage, StorageDestination
 from sematic.utils.memoized_property import memoized_property
 from sematic.utils.retry import retry
 
 logger = logging.getLogger(__name__)
 
-_PLUGIN_VERSION = (0, 1, 0)
+_PLUGIN_VERSION = (0, 2, 0)
```

Comment:
The schema of the plugin itself does not change in the settings file, so this shouldn't be bumped. Please revert.